features/test/een-ouder-staande-huwelijk.feature

```diff
@@ -0,0 +1,153 @@
+      En 'moeder' en 'partner' zijn <datum echtscheiding> gescheiden
+      Als 'gezag' wordt gevraagd van 'minderjarige'
+      Dan is het gezag over 'minderjarige' gezamenlijk gezag met ouder 'moeder' en derde 'partner'
+
+      Voorbeelden:
+        | datum echtscheiding | omschrijving                       |
+        | gisteren - 7 jaar   | ruim na geboorte van minderjarige  |
+        | morgen - 8 jaar     | 1 dag na geboorte van minderjarige |
+
+    Scenario: moeder was nooit gehuwd
+      Gegeven de minderjarige persoon 'minderjarige' met één ouder 'moeder'
+      Als 'gezag' wordt gevraagd van 'minderjarige'
+      Dan is het gezag over 'minderjarige' eenhoofdig ouderlijk gezag met ouder 'moeder'
+
+    Scenario: moeder was gehuwd en is voor de geboorte gescheiden
+      Gegeven de persoon 'partner' met burgerservicenummer '000000024'
+      * is meerderjarig
+      En de minderjarige persoon 'minderjarige' met één ouder 'moeder'
+      * is vandaag - 8 jaar geboren
+      En 'moeder' en 'partner' zijn morgen - 10 jaar gehuwd
+      En 'moeder' en 'partner' zijn <datum echtscheiding> gescheiden
+      Als 'gezag' wordt gevraagd van 'minderjarige'
+      Dan is het gezag over 'minderjarige' eenhoofdig ouderlijk gezag met ouder 'moeder'
+
+      Voorbeelden:
+        | datum echtscheiding | omschrijving                         |
+        | morgen - 9 jaar     | ruim voor geboorte van minderjarige  |
+        | gisteren - 8 jaar   | 1 dag voor geboorte van minderjarige |
+        | vandaag - 8 jaar    | op de geboortedag van minderjarige   |
+
+    Scenario: moeder is gehuwd na geboorte
+      Gegeven de persoon 'partner' met burgerservicenummer '000000024'
+      * is meerderjarig
+      En de minderjarige persoon 'minderjarige' met één ouder 'moeder'
+      * is vandaag - 8 jaar geboren
+      En 'moeder' en 'partner' zijn <huwelijksdatum> gehuwd
+      Als 'gezag' wordt gevraagd van 'minderjarige'
+      Dan is het gezag over 'minderjarige' eenhoofdig ouderlijk gezag met ouder 'moeder'
+
+      Voorbeelden:
+        | huwelijksdatum  | omschrijving                       |
+        | morgen - 6 jaar | ruim na geboorte van minderjarige  |
+        | morgen - 8 jaar | 1 dag na geboorte van minderjarige |
+
+  Regel: Als de minderjarige tijdens het huwelijk of partnerschap van de juridische ouder is geadopteerd, dan hebben de ouder en diens (toenmalige) partner gezamenlijk gezag
+
+  Regel: Als de minderjarige tijdens het huwelijk of partnerschap van de juridische ouder geboren is en er sprake is van ontkenning vaderschap, dan heeft de ouder eenhoofdig ouderlijk gezag
+
+  Regel: Als de ouder is overleden of onbevoegd is voor gezag, dan is er voogdij door de partner
+
+  Regel: Als de partner is overleden of onbevoegd is voor gezag, dan is er eenhoofdig ouderlijk gezag
+
+  Regel: Als zowel de ouder als de partner is overleden of onbevoegd is voor gezag, dan is er tijdelijk geen gezag
```

Comment:
Deze regels zijn leeg. Is dat de bedoeling? Zo ja, wellicht een kleine comment (met de waarom) zodat het duidelijk is dat het bewust leeg is gelaten.

Reply:
ik heb opmerking toegevoegd